Meadowlark.net/Meadowlark.Net.Core/Handler/FrontendFacade.cs

```diff
@@ -0,0 +1,37 @@
+using Meadowlark.Net.Core.Model;
+using Meadowlark.Net.Core.Utility;
+using static Meadowlark.Net.Core.Middleware.ApiSchemaLoadingMiddleware;
+using static Meadowlark.Net.Core.Middleware.ParsePathMiddleware;
+using static Meadowlark.Net.Core.Middleware.ValidateEndpointMiddleware;
+using static Meadowlark.Net.Core.Middleware.ValidateDocumentMiddleware;
+
+namespace Meadowlark.Net.Core;
+
+
+public static class FrontendFacade
```

Comment:
Some of these static classes will need to become instance classes so that they can implement interfaces and thus be used in dependency injection... thereby facilitating mocking in unit tests.